apps/web/pages/event-types/[type].tsx

```diff
@@ -1181,6 +1192,106 @@ const EventTypePage = (props: inferSSRProps<typeof getServerSideProps>) => {
 
                       <hr className="border-neutral-200" />
 
+                      <div className="block sm:flex">
+                        <div className="min-w-48 mb-4 sm:mb-0">
+                          <label
+                            htmlFor="bufferTime"
+                            className="mt-2.5 flex text-sm font-medium text-neutral-700">
+                            {t("buffer_time")}
+                          </label>
+                        </div>
+                        <div className="w-full">
+                          <div className="inline-flex w-full space-x-2">
+                            <div className="w-full">
+                              <label
+                                htmlFor="beforeBufferTime"
+                                className="mb-2 flex text-sm font-medium text-neutral-700">
+                                {t("before_event")}
+                              </label>
+                              <Controller
+                                name="beforeBufferTime"
+                                control={formMethods.control}
+                                render={() => {
+                                  const beforeBufferOptions = [
+                                    {
+                                      label: t("event_buffer_default"),
+                                      value: 0,
+                                    },
+                                    ...[5, 10, 15, 20, 30, 45, 60].map((minutes) => ({
+                                      label: minutes + " " + t("minutes"),
+                                      value: minutes,
+                                    })),
+                                  ];
+                                  return (
+                                    <Select
+                                      isSearchable={false}
+                                      classNamePrefix="react-select"
+                                      className="react-select-container focus:border-primary-500 focus:ring-primary-500 block w-full min-w-0 flex-1 rounded-sm border border-gray-300 sm:text-sm"
+                                      onChange={(val) => {
```

Comment:
The way you are currently doing it is actually not requiring Controller; as you still talk to the form manually and not setting up the Select for being controlled.

Have a look at:
```javascript
<Controller
  /* ... */
  render={({ field: { onChange, onBlur, value, ref } }) => (
    /* ... */
    return (
      <Select
        /* ... */
        onChange={(val) => {
          if (val) onChange(val.value);
          // or onChange(val!.value);
        }

        // as well as instead of having to use eventType.beforeEventBuffer
        beforeBufferOptions.find( (option) => option.value === value );
        // now you can pass the value in the Controller instead of the <Select> itself.

Reply:
Oh right 🤦‍♂️. I'll get this sorted. Thanks 😄